apps/api/lib/email/tp-templates/company-profile-approved.mjml

```diff
@@ -17,29 +17,29 @@
           <mj-text mj-class="text" padding="0 0 20px 0"
             >Congratulations - your company profile has been approved!</mj-text
           >
-          <mj-text mj-class="text" padding="0 0 20px 0"
+          <mj-text mj-class="text" padding="0 0 20px 0" align="justify"
```

Comment:
Do we have a good reason to add justified text alignment here and in all the other emails? We might want to consider updating _all_ emails to have justified text. Should we consult the designers?

Reply:
Here is the same reason as above. I would update _all_ emails to have justified text.
Sorry for too many unauthorized changes:) Should I consult the designers?

Without text alignment:
![image](https://user-images.githubusercontent.com/51786805/201358554-312b7bf4-c410-4996-986e-d65c989b7391.png)

With text alignment:
![image](https://user-images.githubusercontent.com/51786805/201359480-f61a31a2-26ea-4490-ae57-1e30f8ac2f04.png)

Reply:
I'm ashamed how we didn't think of this before. It makes a huge change. It looks way more professional like this. Great job seeing this @katamatata

Reply:
Agreed, lots of kudos to @katamatata

Reply:
This is the way! :grin:

Reply:
[This is the way](https://www.youtube.com/watch?v=xtU7VhPOGLc)